rust/lance-core/src/encodings/plain.rs

```diff
@@ -748,198 +741,121 @@ mod tests {
-            file_writer
-                .write(&[
-                    RecordBatch::try_new(arrow_schema.clone(), vec![Arc::new(slice)]).unwrap(),
-                ])
-                .await
-                .unwrap();
-        }
-        file_writer.finish().await.unwrap();
-
-        let batch = read_file_as_one_batch(&store, &path).await;
-        assert_eq!(
-            batch.column_by_name("fl").unwrap().as_ref(),
-            &fixed_size_list
-        );
-    }
-
-    #[tokio::test]
-    async fn test_take_boolean() {
-        let store = ObjectStore::memory();
-        let path = Path::from("/take_bools");
-
-        let arrow_schema = Arc::new(ArrowSchema::new(vec![Field::new(
-            "b",
-            DataType::Boolean,
-            false,
-        )]));
-        let schema = Schema::try_from(arrow_schema.as_ref()).unwrap();
-        let mut file_writer =
-            FileWriter::try_new(&store, &path, schema.clone(), &Default::default())
-                .await
-                .unwrap();
-
-        let array = BooleanArray::from((0..120).map(|v| v % 5 == 0).collect::<Vec<_>>());
-        let batch =
-            RecordBatch::try_new(arrow_schema.clone(), vec![Arc::new(array.clone())]).unwrap();
-        file_writer.write(&[batch]).await.unwrap();
-        file_writer.finish().await.unwrap();
-
-        let reader = FileReader::try_new(&store, &path).await.unwrap();
-        let actual = reader
-            .take(&[2, 4, 5, 8, 63, 64, 65], &schema)
-            .await
-            .unwrap();
-
-        assert_eq!(
-            actual.column_by_name("b").unwrap().as_ref(),
-            &BooleanArray::from(vec![false, false, true, false, false, false, true])
-        );
-    }
-
-    #[tokio::test]
-    async fn test_take_boolean_beyond_chunk() {
-        let mut store = ObjectStore::memory();
-        store.set_block_size(256);
-        let path = Path::from("/take_bools");
-
-        let arrow_schema = Arc::new(ArrowSchema::new(vec![Field::new(
-            "b",
-            DataType::Boolean,
-            false,
-        )]));
-        let schema = Schema::try_from(arrow_schema.as_ref()).unwrap();
-        let mut file_writer =
-            FileWriter::try_new(&store, &path, schema.clone(), &Default::default())
-                .await
-                .unwrap();
-
-        let array = BooleanArray::from((0..5000).map(|v| v % 5 == 0).collect::<Vec<_>>());
-        let batch =
-            RecordBatch::try_new(arrow_schema.clone(), vec![Arc::new(array.clone())]).unwrap();
-        file_writer.write(&[batch]).await.unwrap();
-        file_writer.finish().await.unwrap();
-
-        let reader = FileReader::try_new(&store, &path).await.unwrap();
-        let actual = reader.take(&[2, 4, 5, 8, 4555], &schema).await.unwrap();
-
-        assert_eq!(
-            actual.column_by_name("b").unwrap().as_ref(),
-            &BooleanArray::from(vec![false, false, true, false, true])
-        );
-    }
+    // #[tokio::test]
```

Comment:
why are these disabled?

Reply:
This requires `FileReader / FileWriter`, which have not been migrated yet.

Reply:
I guess we need to finish the migration before next release?

Reply:
it is ok, no user facing changes.  The pub structure / traits are re-exported in `lance`